code/modules/clothing/under/skirt_dress.dm

```diff
@@ -27,17 +46,35 @@
 	body_parts_covered = CHEST|GROIN|LEGS
 	flags_inv = HIDESHOES
 
-/obj/item/clothing/under/dress/redeveninggown
```

Comment:
Did any of the maps have the evening gown in them? I always sucked at reading failed checks but it seems they're mostly map errors with creating/destroying things that aren't there.

Reply:
I'll check when I'm home on the weekend, away for work for a few more days.

Reply:
Delta has one

Reply:
Noted! Thank you. I'll go through all the .dmm files and check tomorrow too.